src/hotspot/os/posix/vmError_posix.cpp

```diff
@@ -148,7 +141,6 @@ void VMError::reset_signal_handlers() {
     os::signal(SIGNALS[i], CAST_FROM_FN_PTR(void *, crash_handler));
     sigaddset(&newset, SIGNALS[i]);
   }
-  PosixSignals::unblock_thread_signal_mask(&newset);
```

Comment:
AFAICS newset is now unused with this change.

Reply:
You are right, I can remove more.